cypress/integration/01-Startup.spec.js

```diff
@@ -0,0 +1,15 @@
+describe("Application", () => {
```

Comment:
Can we rename it to something else i.e. `basic.spec.js`?

Reply:
Yes, sure

Reply:
done